### DISCUSSION TITLE
The following environment variables are being overriden by the config file

### DISCUSSION THREAD

**Discussion by art-lucas** [dani-garcia/vaultwarden#1281] [General]
This is probably not an issue. When starting bitwarden_rs (`systemctl start bitwarden`, `systemctl status bitwarden`), these warning messages are displayed:

```
[WARNING] The following environment variables are being overriden by the config file,
[WARNING] please use the admin panel to make changes to them:
[WARNING] DOMAIN, SIGNUPS_ALLOWED, SIGNUPS_VERIFY, INVITATIONS_ALLOWED, PASSWORD_ITERATIONS, ADMIN_TOKEN
```
Are these lines informative only, i.e., can they be ignored?

**Comment by BlackDex** [comment]
It shows which items are different between the environment variables/.env file and the saved config when you have modified and saved it via the admin interface.

So you can ignore it if all is working.

**Comment by art-lucas** [comment]
Thanks, it makes sense. Which config is read/used last, the .env file or the settings from the admin panel?

**Reply by gramakri** [reply]
@art-lucas Settings from admin panel override the env file
